spec/support/spec_config.rb

```diff
@@ -410,17 +410,21 @@ def auth_options
   end
 
   def ssl_options
-    if ssl?
-      {
+    return {} unless ssl?
+    {
         ssl: true,
         ssl_verify: true,
-        ssl_cert:  client_cert_path,
-        ssl_key:  client_key_path,
-        ssl_ca_cert: ca_cert_path,
-      }.merge(Utils.underscore_hash(@uri_tls_options))
-    else
-      {}
-    end
+    }.tap do |options|
+      if @mongodb_uri.servers.grep(/mongodb.*\.net/).empty?
```

Comment:
The domain is `mongodb.net` right? So that should be matched verbatim?

Reply:
Apparently, test serverless instances are in the domain `mongodb-dev.net`, hence the regex.

Reply:
Can you add that as a comment please.

Reply:
Done.